tests/playwright/pages/widgets/google-maps.ts

```diff
@@ -1,21 +1,27 @@
-import EditorSelectors from '../../selectors/editor-selectors';
 import Content from '../elementor-panel-tabs/content';
 
 export default class GoogleMaps extends Content {
-	async setGoogleMapsParams( args : { location: string, zoom: string, height: string } ) {
```

Comment:
Why remove this method?

Reply:
@yotamselementor The line of thought was different. Helper functions are typically created for processes that are executed over and over in various locations. It is simpler to use and maintain reusable methods.

In this instance, though, we wrap three lines in a function and use it just once. There's no need for that. Otherwise, we will start groping and wrapping all of our code.

Reply:
Extracting this to a separate function (could be in the test file) makes things more readable though

Reply:
The 3 lines setting the control values are readable too. Moving 3 lines to an external function doesn't make it un-readable.

Reply:
PR updated.